fellow-submissions/stephanie-smith/index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/blocks.css/dist/blocks.min.css" />
+    <title>Quiz Time!</title>
+  </head>
+  
+  <body>
+    <h1>Music Theory Quiz!</h1>
+    <p>Hello, and welcome to my quiz! The topic of this quiz is music theory, but it's alright if you don't know much. Hopefully you learn
+    something from it! Enjoy!</p>
+    
+    <form action="test" method="post">
+      <fieldset class="block fixed color themed">
+        <legend>Question #1</legend>
+        <label for="First Question">Let's start easy. <strong>What note lasts 2 beats?</strong></label>
+        <ul>
+          <li><input type="radio" name="note" value="whole">Whole note</li>
+          <li><input type="radio" name="note" value="quarter">Quarter note</li>
+          <li><input type="radio" name="note" value="half">Half note</li>
+          <li><input type="radio" name="note" value="eighth">Eighth note</li>
+          </ul>
+      </fieldset>
+      
+      <fieldset class="block fixed color themed">
+        <legend>Question #2</legend>
+        <label for="Second Question">Now, <strong>what is the time signature 2/2 also known as?</strong> Check all that apply!</label>
+        <ul>
+          <li><input type="checkbox" name="name" value="cut">Cut time</li>
+          <li><input type="checkbox" name="name" value="fast">Fast time</li>
+          <li><input type="checkbox" name="name" value="alla">Alla breve</li>
+          <li><input type="checkbox" name="name" value="common">Common time</li>
+          <li><input type="checkbox" name="name" value="half">Half time</li>
+        </ul>
+      </fieldset>
+      
+      <fieldset class="block fixed color themed">
+        <legend>Question #3</legend>
+        <label for="Third Question">Here's something a little more difficult. <strong>In your own terms, what is syncopation?</strong></label>
+        <textarea placeholder="Put your answer here!"></textarea>
+      </fieldset>
+      
+      <fieldset class="block fixed color themed">
+        <legend>Question #4</legend>
+        <label for="Fourth Question">If a scale has one flat, <strong>what is it called?</strong></label>
+        <input type="text" placeholder="Put your answer here!" name="letter">
+      </fieldset>
+      
+      <fieldset class="block fixed color themed">
+        <legend>Question #5</legend>
+        <label for="Fifth Question">Lastly, <strong>what is one of the well-known acronyms for remembering all the major scales?</strong> There's more than one answer you can get right!</label>
+        <select name="acros">
+          <option value="select">Please make a selection~</option>
```

Comment:
We don't want to include this placeholder as an `<option>` because that technically means that a user can select it. Can you find out how to use a placeholder with `<select>`